WordPressAuthenticator/Analytics/AnalyticsTracker.swift

```diff
@@ -229,25 +233,24 @@ public class AnalyticsTracker {
         }
     }
     
-    let context: Context
+    private let state = State()
     
     /// The backing analytics tracking method.  Can be overridden for testing purposes.
     ///
     let track: TrackerMethod
-    
+
     // MARK: - Initializers
     
-    init(context: Context, track: @escaping TrackerMethod = WPAnalytics.track) {
-        self.context = context
+    init(track: @escaping TrackerMethod = WPAnalytics.track) {
         self.track = track
     }
     
     // MARK: - Tracking
     
     /// Track a step within a flow.
     ///
-    func track(step: Step, flow: Flow) {
-        track(event(step: step, flow: flow))
+    func track(step: Step) {
```

Comment:
The flow is now set explicitly when it changes.  No need to specify it for each call.

This ensures that we can log a step even without knowing precisely what flow we're in.  This is particularly useful for loggin in the success in our TwoFactorVC, since it's a success regardless of what flow we're in.